### PR TITLE
EWPP-4269: Replace hardcoded webtools url.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by nagyad — APPROVED** [review]

**Review by fmmribeiro — APPROVED** [review]

File: tests/fixtures/static_html/etrans_button.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>A page containing an eTrans button</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```
Comment (alexboss) [inline tests/fixtures/static_html/etrans_button.html:6]: The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

File: tests/fixtures/static_html/etrans_icon.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>A page containing an eTrans button</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```
Comment (alexboss) [inline tests/fixtures/static_html/etrans_icon.html:6]: The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

File: tests/fixtures/static_html/etrans_link.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>A page containing an eTrans button</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```
Comment (alexboss) [inline tests/fixtures/static_html/etrans_link.html:6]: The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

File: tests/fixtures/static_html/one_map.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>A page containing 1 map</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```
Comment (alexboss) [inline tests/fixtures/static_html/one_map.html:6]: The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

File: tests/fixtures/static_html/two_maps.html [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>A page containing 1 map</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```
Comment (alexboss) [inline tests/fixtures/static_html/two_maps.html:6]: The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

File: tests/src/Functional/SmartLoaderDependenciesTest.php [hunk 6]
```diff
@@ -48,7 +48,7 @@ public function testLibraryLoading(string $module, string $url, $setup_method =
     $this->container->get('kernel')->invalidateContainer();
 
     $this->drupalGet($url);
-    $this->assertSession()->responseContains('<script src="https://europa.eu/webtools/load.js');
+    $this->assertSession()->responseContains('<script src="https://webtools.europa.eu/load.js');
```
Comment (alexboss) [inline tests/src/Functional/SmartLoaderDependenciesTest.php:51]: Unless the `defer` attribute is added later in the code logic, we would recommend to add the `defer` attribute.
The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets

**Comment by julien-** [comment]
Hello @nagyad,
Why not 2.0.0 ? It is a blocker update

File: modules/oe_webtools_captcha/config/install/oe_webtools_captcha.settings.yml [hunk 7]
```diff
@@ -1 +1 @@
-validationEndpoint: 'https://europa.eu/webtools/rest/captcha/verify'
+validationEndpoint: 'https://webtools.europa.eu/rest/captcha/verify'
```
Comment (alexboss) [inline modules/oe_webtools_captcha/config/install/oe_webtools_captcha.settings.yml:1]: For information, please check https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=1851981836 especially **§ Communication sent on April 14th, 2024**  for possible impacts.

File: modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php [hunk 8]
```diff
@@ -154,7 +154,7 @@ public function createSourceField(MediaTypeInterface $type) {
 
     $generator_link = Link::fromTextAndUrl(
       $this->t('Webtools wizard'),
-      Url::fromUri('https://europa.eu/webtools/mgmt/wizard/')
+      Url::fromUri('https://webtools.europa.eu/mgmt/wizard/')
```
Comment (alexboss) [inline modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php:157]: The URL of the Wizard is now https://webtools.europa.eu/tools/#/wizards

**Review by alexboss — COMMENTED** [review]
Dear colleagues,
To whom it may concern.
Sorry to step in, Alexandre from DIGIT WEBTOOLS. Got this link from a Drupal Dev on a Teams channel.
I've added some feedback from Webtools point of view.
Thank you and best regards.
Alexandre - DIGIT WEBTOOLS

**Comment by nagyad** [comment]
Thank you for the review @alexboss! We have evaluated each one of your comments, and we didn't find a blocking issue. The relevant ones will be handled in follow-ups. 

@julien- As there is no API change and there are redirects in place by devops to the URLs in question, there is no need for a major release.

File: tests/fixtures/static_html/etrans_button.html (hunk 1 above)
Comment (nagyad) [reply tests/fixtures/static_html/etrans_button.html:6]: These are fixtures for tests and the library in the Drupal website is included with defer as per https://github.com/openeuropa/oe_webtools/blob/master/oe_webtools.libraries.yml#L5

File: tests/fixtures/static_html/etrans_icon.html (hunk 2 above)
Comment (nagyad) [reply tests/fixtures/static_html/etrans_icon.html:6]: These are fixtures for tests and the library in the Drupal website is included with defer as per https://github.com/openeuropa/oe_webtools/blob/master/oe_webtools.libraries.yml#L5

File: tests/fixtures/static_html/etrans_link.html (hunk 3 above)
Comment (nagyad) [reply tests/fixtures/static_html/etrans_link.html:6]: These are fixtures for tests and the library in the Drupal website is included with defer as per https://github.com/openeuropa/oe_webtools/blob/master/oe_webtools.libraries.yml#L5

File: tests/fixtures/static_html/two_maps.html (hunk 5 above)
Comment (nagyad) [reply tests/fixtures/static_html/two_maps.html:6]: These are fixtures for tests and the library in the Drupal website is included with defer as per https://github.com/openeuropa/oe_webtools/blob/master/oe_webtools.libraries.yml#L5

File: modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php (hunk 8 above)
Comment (nagyad) [reply modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php:157]: This seems relevant.

File: modules/oe_webtools_captcha/config/install/oe_webtools_captcha.settings.yml (hunk 7 above)
Comment (nagyad) [reply modules/oe_webtools_captcha/config/install/oe_webtools_captcha.settings.yml:1]: This can be overridden by an environent variable.

**Pull request merged**